website/docs/sdk-reference/dart.md

```diff
@@ -102,6 +104,32 @@ final value = await client.getValue(
 );
 ```
 
+## Anatomy of `getValueDetails()`
+
+`getValueDetails()` is similar to `getValue()` but instead of returning only the evaluated value it gives more detailed information about the evaluation result.
+
+| Parameters     | Description                                                                                                  |
+| -------------- | ------------------------------------------------------------------------------------------------------------ |
+| `key`          | **REQUIRED.** The key of a specific setting or feature flag. Set on *ConfigCat Dashboard* for each setting.                           |
+| `defaultValue` | **REQUIRED.** This value will be returned in case of an error.                                               |
+| `user`         | Optional, *User Object*. Essential when using Targeting. [Read more about Targeting.](advanced/targeting.md) |
+```dart
+final details = await client.getValueDetails(
+    key: 'keyOfMySetting', 
+    defaultValue: false,
+    user: ConfigCatUser(identifier: '#USER-IDENTIFIER#'), // Optional User Object
+);
+
+// Details holds the following information:
+details.value;                              // The evaluated value of the feature flag or setting.
+details.key;                                // The key of the evaluated feature flag or setting.
+details.isDefaultValue;                     // True when the default value passed to getValueDetails() is returned due to an error.
+details.error;                              // In case of an error this field will contain its message.
+details.matchedEvaluationPercentageRule;    // When the evaluation was based on a percentage rule, this field will contain that specific rule.
```

Comment:
If the evaluation was based on a percentage rule, this field contains that specific rule.

Reply:
✅